docs/content/reference/exploiters/powershell.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "PowerShell"
+date: 2021-08-24T12:19:21+03:00
+draft: false
+tags: ["exploit", "windows"]
+---
+
+### Description
+
+PowerShell Remoting is a brute-force exploit that uses PowerShell Remoting Protocol (PSRP) and
+Windows Remote Management(WinRM) services to propagate to a victim.
```

Comment:
I'm not sure if this is 100% accurate. PSRP is a protocol and PowerShell Remoting is a method of controlling Windows systems. To say that this is an exploit that uses PSRP may suggest that we're exploiting the protocol, which is false.

Try something like this (this is not the best but all that I could come up with right now):
```suggestion
This exploiter uses brute-force to propagate to a victim through PowerShell Remoting using Windows Remote Management (WinRM).
```

Reply:
I don't think we need to be 100% accurate or precise. We're exploiting remote powershell setups, we don't need to elaborate on which protocols powershell remoting uses, right?

Reply:
Exactly, no need to mention PSRP at all.